crates/bevy_ecs/src/component.rs

```diff
@@ -596,8 +596,10 @@ impl Tick {
     }
 
     #[inline]
-    /// Returns `true` if the tick is older than the system last's run.
-    pub fn is_older_than(&self, last_change_tick: u32, change_tick: u32) -> bool {
+    /// Returns `true` if the tick occurred since the system's `last_change_tick`.
+    ///
+    /// Uses the system's current `change_tick` as a reference to help deal with wraparound.
```

Comment:
```suggestion
    /// Uses the system's current change tick as a reference to help deal with wraparound.
```

Writing `change_tick` like that is misleading, since it looks like you're referring to the parameter `change_tick`.

Reply:
I am referring to the parameter.

Maybe "`change_tick` is the current tick of the system, used as a reference to help deal with wraparound" would be more clear?

Reply:
Yes, that helps significantly.

Reply:
I think your original phrasing is clear enough, I just misunderstood the code that it was describing :)

Reply:
I updated the wording.